src/main/cpp/vision/definitions/CameraConfig_302.cpp

```diff
@@ -38,7 +38,7 @@ void CameraConfig_302::BuildCameraConfig()
                                                  DRAGON_LIMELIGHT_CAM_MODE::CAM_VISION         // CAM_MODE camMode,
 
     ); // additional parameter
-    DragonVision::GetDragonVision()->AddLimelight(front, DRAGON_LIMELIGHT_CAMERA_USAGE::APRIL_TAGS);
+    DragonVision::GetDragonVision()->AddLimelight(front, DRAGON_LIMELIGHT_CAMERA_USAGE::BOTH);
```

Comment:
We should add usage for algae and algae_and_apriltags, then when we switch to drive to algae, we can only change the cameras that it is valid for.  Maybe both should go away.